[Focal] LTS channel allowed for upgrades

## Description

The release management settings for Focal permit `do-release-upgrade` to the next LTS version. There isn't one of those yet, but tests are failing because of it:

```
        def test_release_manager_upgrade_channel(host):
            """
            Ensures that the `do-release-upgrade` command will not
            suggest upgrades from Xenial to Bionic (which is untested
            and unsupported.)
            """
            expected_channels = {
                "xenial": "never",
                "focal": "never",
            }
    
            config_path = "/etc/update-manager/release-upgrades"
            assert host.file(config_path).is_file
    
            raw_output = host.check_output("grep '^Prompt' {}".format(config_path))
            _, channel = raw_output.split("=")
    
            expected_channel = expected_channels[host.system_info.codename]
    >       assert channel == expected_channel
    E       AssertionError: assert 'lts' == 'never'
    E         - never
    E         + lts
```

To see that test fail, you'll need to use hardware or Qubes VMs (installed from ISO), since the bento boxes modify the setting https://github.com/chef/bento/blob/3dd165cf422572ab4240c8f1ce9a7e40b7960457/packer_templates/ubuntu/scripts/update.sh#L5 in a similar fashion to what we implemented for Xenial in https://github.com/freedomofpress/securedrop/pull/3546/files

## Comments

This is almost certainly a duplicate of #5781, but filing separately until that's confirmed.

--- a/install_files/securedrop-config-focal/DEBIAN/postinst
+++ b/install_files/securedrop-config-focal/DEBIAN/postinst
@@ -4,8 +4,17 @@
 set -e
 set -x
 
+# Issue #5782
+# Set Prompt=never for distro upgrades
+update_release_prompt() {
+    set -e
+    upgrade_config='/etc/update-manager/release-upgrades'
+    sed -i 's/Prompt=.*/Prompt=never/' "$upgrade_config"
+}
+
 case "$1" in
     configure)
+    update_release_prompt
     # Configuration required for unattended-upgrades
     cp /opt/securedrop/20auto-upgrades /etc/apt/apt.conf.d/
     cp /opt/securedrop/50unattended-upgrades /etc/apt/apt.conf.d/

